usr/share/rear/layout/save/FDRUPSTREAM/Linux-s390/990_copy_disklayout_file.sh

```diff
@@ -24,6 +24,7 @@ opath=$( output_path $scheme $path )
 if [[ "$ZVM_NAMING" == "Y" && "$ARCH" == "Linux-s390" ]] ; then 
       VM_UID=$(vmcp q userid |awk '{ print $1 }')
       LogPrint "s390 dislayout.conf will be saved as $(opath)/$VM_UID.disklayout.conf"
+      mkdir -p ${opath}/"$VM_UID"
```

Comment:
The command
```
LogPrint "s390 dislayout.conf will be saved as $(opath)/$VM_UID.disklayout.conf"
```
cannot work because `$(opath)` is command substitution as follows:
```
# opath=/tmp//path/to/somewhere

# VM_UID=vm-UID

# echo "s390 dislayout.conf will be saved as $(opath)/$VM_UID.disklayout.conf"
opath: command not found
s390 dislayout.conf will be saved as /vm-UID.disklayout.conf
```
Usually you do not need the complicated `${var1}/${var2}` syntax
unless the character in between is one of the allowed characters
in a bash variable name, cf. "man bash"
```
word
  A sequence of characters considered
  as a single unit by the shell.
  Also known as a token.

name
  A word consisting only of alphanumeric characters
  and underscores, and beginning with an alphabetic
  character or an underscore.
  Also referred to as an identifier.
```
So `$opath/$VM_UID` is simpler and in the end more fail safe.
Accordingly I would suggest to do it this way:
```
VM_UID=$( vmcp q userid |awk '{ print $1 }' )
LogPrint "s390 dislayout.conf will be saved as $opath/$VM_UID.disklayout.conf"
mkdir -p $opath/$VM_UID
```

That seems to work at least for me:
```
# opath=/tmp/path/to/somewhere

# VM_UID=vm-UID

# echo "s390 dislayout.conf will be saved as $opath/$VM_UID.disklayout.conf"
s390 dislayout.conf will be saved as /tmp/path/to/somewhere/vm-UID.disklayout.conf

# mkdir -vp $opath/$VM_UID
mkdir: created directory '/tmp/path'
mkdir: created directory '/tmp/path/to'
mkdir: created directory '/tmp/path/to/somewhere'
mkdir: created directory '/tmp/path/to/somewhere/vm-UID'
```